src/olympia/scanners/admin.py

```diff
@@ -229,26 +241,38 @@ def get_urls(self):
                 self.admin_site.admin_view(self.handle_true_positive),
                 name='scanners_scannerresult_handletruepositive',
             ),
+            url(
+                r'^(?P<pk>.+)/revert-report/$',
+                self.admin_site.admin_view(self.handle_revert),
+                name='scanners_scannerresult_handlerevert',
+            ),
         ]
         return custom_urls + urls
 
     def result_actions(self, obj):
-        if not obj.can_report_feedback():
-            return
-
-        return format_html(
-            '<a class="button" href="{}">Report as false positive</a>'
-            '&nbsp;'
-            '<a class="button default" href="{}">Mark as true positive</a>',
-            reverse(
-                'admin:scanners_scannerresult_handlefalsepositive',
-                args=[obj.pk],
-            ),
-            reverse(
-                'admin:scanners_scannerresult_handletruepositive',
-                args=[obj.pk],
-            ),
-        )
+        if obj.can_report_feedback():
```

Comment:
This makes me think that we should likely use a `<form>` and a `POST` method instead of using `GET` to update things. Not something to change in this patch, though.

Reply:
Yes, I agree with both. I wanted to have a quick solution to make it into the next tag. Unfortunately, it'll have to wait a little longer now, because we don't push this week.

Reply:
I'll file an issue :)